src/Compilers/.editorconfig

```diff
@@ -1,4 +1,8 @@
 # Project-specific settings:
+[*.{cs,vb}]
+# Local functions are camelCase
+dotnet_naming_style.local_function_style.capitalization = camel_case
```

Comment:
This might deserve a comment in the parent .editorconfig. If somebody "helpfully" renames the local_function_style to something else, this override might silently disappear. Hopefully that creates warnings, but...

Reply:
Alternately, this is a new feature, can't we just agree on this one?

Reply:
> can't we just agree on this one?

I'll submit a follow-up issue to try and agree on it, but my understanding is the current form matches the currently-used preferences.